src/System.Security.Principal.Windows/tests/System.Security.Principal.Windows.Tests.builds

```diff
@@ -4,7 +4,7 @@
   <ItemGroup>
     <Project Include="System.Security.Principal.Windows.Tests.csproj">
       <OSGroup>Windows_NT</OSGroup>
-      <TestTFMs>netcore50;netcoreapp1.0;net46;netcoreapp;net463</TestTFMs>
+      <TestTFMs>netcore50;netcoreapp1.0;net46;netcoreapp1.1;net463</TestTFMs>
```

Comment:
@weshaggard I changed this when I merged with the master (after the dev-eng work)... `netcoreapp1.1` seemed like the more logical choice, am I right?